src/app/utils/testing.utils.ts

```diff
@@ -217,3 +228,155 @@ function mockPlates(args?: Partial<Plate>): Plate {
+    letterContents: 'some letter text'
+  };
+  return { ...mock, ...args };
+}
+
+function mockPurchaser(args?: Partial<PurchaserDetails>): PurchaserDetails {
+  const mock: PurchaserDetails = {
+    name: 'Ben',
+    address1: 'Garrets green',
+    address2: 'Birmingham',
+    address3: 'lala land',
+    postCode: 'NG4 12Z',
+    postTown: 'A town',
+    telephoneNumber: '123123123',
+    emailAddress: 'test@test.com',
+    faxNumber: '1231234',
+    purchaserNotes: 'some notes'
+  };
+  return { ...mock, ...args };
+}
+
+function mockManufacturer(args?: Partial<ManufacturerDetails>): ManufacturerDetails {
+  const mock: ManufacturerDetails = {
+    name: 'Ben',
+    address1: 'Garrets green',
+    address2: 'Birmingham',
+    address3: 'lala land',
+    postCode: 'NG4 12Z',
+    postTown: 'A town',
+    telephoneNumber: '123123123',
+    emailAddress: 'test@test.com',
+    faxNumber: '1231234',
+    manufacturerNotes: 'some notes'
+  };
+  return { ...mock, ...args };
+}
+
+
+function mockTechRecHistory(args?: Partial<VehicleTechRecordModel>): VehicleTechRecordModel {
+  const tecRec: TechRecord = {
+                              statusCode: 'current',
+                              reasonForCreation: 'reason for creation',
+                              createdByName: 'test name',
+                              createdAt: '1-1-2020',
+                              lastUpdatedByName: 'test 1',
+                              lastUpdatedAt: '12-3-2020'
+                             } as TechRecord;
+  const mock: VehicleTechRecordModel = {techRecord: [tecRec]} as VehicleTechRecordModel;
+  return { ...mock, ...args };
+}
+
+function mockVehicleSummary(args?: Partial<TechRecord>): TechRecord {
+  const axles: Axle = { parkingBrakeMrk: false, axleNumber: 1} as Axle;
+  const mock: TechRecord = {
+    vehicleSize: 'small',
+    vehicleType: 'psv',
+    axles: [axles],
+    approvalType: 'NTA',
+    approvalTypeNumber: 'string',
+    ntaNumber: 'string',
+    variantNumber: '22',
+    variantVersionNumber: '3435',
+    fuelPropulsionSystem: 'DieselPetrol',
+    offRoad: true,
+    numberOfWheelsDriven: 8,
+    euVehicleCategory: 'm2',
+    emissionsLimit: 5,
+    departmentalVehicleMarker: false,
+    alterationMarker: true
+  } as TechRecord;
+  return { ...mock, ...args };
+}
+
+function mockWeightsTrl(args?: Partial<TechRecord>): TechRecord {
+  const mock = {
+    'vehicleType': 'trl',
+    'grossGbWeight': 3,
+    'grossDesignWeight': 2,
+    'grossEecWeight': 3,
+    'axles': [
+      {
+        'axleNumber': 1,
+        'weights': {
+          'gbWeight': 500,
+          'eecWeight': 500,
+          'designWeight': 500
+        }
+      },
+      {
+        'axleNumber': 2,
+        'weights': {
+          'gbWeight': 500,
+          'eecWeight': 500,
+          'designWeight': 500
+        }
+      }
+    ]
+  } as TechRecord;
+
+  return { ...mock, ...args };
+}
+
+function mockWeightsHgv(args?: Partial<TechRecord>): TechRecord {
```

Comment:
there is an existing mockTechRecord method. We should only extend that. We  need a separate mock method of axles